indico/htdocs/sass/modules/_users.scss

```diff
@@ -199,6 +199,10 @@
                 }
             }
 
+            &.placeholder {
+                width: 22px;
+            }
+
```

Comment:
I guess you won't need this if you fix the `if` [here](https://github.com/indico/indico/pull/1890/files#r30422038)